src/index.jsx

```diff
@@ -18,10 +18,13 @@ class App extends React.Component {
         this.saveTimer = 0
     }
 
-    handleThemeChange(theme) {
-        this.saveTheme(theme)
+    handleThemeChange(name, color) {
```

Comment:
I think it's better to rename to `handleColorChange` rather than `handleThemeChange`.
(and 'onThemeChange` to `onColorChange` as well)

How do you think?